grocerylist.py

```diff
@@ -58,8 +58,10 @@ def existing_lst():
 
 #Promt user to enter grocery list
 def user_lst():
```

Comment:
According to the PEP 8 Python Style guide, there are actually recommended ways to write comments! 

I'd suggest making one more commit and follow this guide: 

https://realpython.com/python-pep8/#inline-comments

In short:
* Block comments, describing a section of code, should go right under the function or above the code 
* Inline comments explaining a single line should be added at the end of the line 

```python
def add_one(input_list):
    # this block comment explains how the function add_one works
    # it loops over the input, adds 1 to each value, and returns that new list

    output_list = [] # this inline comment explains that I'm initializing an empty array to return later

    # this block comment explains that my loop below and will take the variable x and add 1 to each value
    for item in input_list:
        item += 1
        output_list.append(item) # this inline comment explains I'm adding the item to the end of the new list
        
    return output_list
```

Could you try reviewing your code and making one more commit to reflect this section of the style guide?